aspnetcore/fundamentals/logging/index.md

```diff
@@ -924,6 +924,33 @@ If the `traceparent` http request header is set, the `ParentId` in the log scope
 
 ::: moniker-end
 
+::: moniker range=">= aspnetcore-6.0"
+
+## Automatically log scope with SpanId, TraceId, ParentId, Baggage, and Tags.
+
+The logging libraries implicitly create a scope object with `SpanId`, `TraceId`, `ParentId`,`Baggage`, and `Tags`. This behavior is configured via <xref:Microsoft.Extensions.Logging.LoggerFactoryOptions.ActivityTrackingOptions>.
+
+```csharp
+  var loggerFactory = LoggerFactory.Create(logging =>
+  {
+      logging.Configure(options =>
+      {
+          options.ActivityTrackingOptions = ActivityTrackingOptions.SpanId
+                                              | ActivityTrackingOptions.TraceId
+                                              | ActivityTrackingOptions.ParentId
+                                              | ActivityTrackingOptions.Baggage
```

Comment:
This looks good to me. Is there a way to tell 

```
 ActivityTrackingOptions.SpanId
| ActivityTrackingOptions.TraceId
| ActivityTrackingOptions.ParentId
```

are supported on .NET 5.0 and 

```
| ActivityTrackingOptions.Baggage
| ActivityTrackingOptions.Tags;
```

supported on .NET 6.0.

I am seeing `moniker range=">= aspnetcore-6.0"` which the user may think they cannot use SpanId, TraceId, and ParentId on .NET 5.0.

Reply:
Take a look at [Internal review URL](https://review.docs.microsoft.com/en-us/aspnet/core/fundamentals/logging/?view=aspnetcore-5.0&branch=pr-en-us-22260#apply-log-filter-rules-in-code) and use the version selector
Use the version selector:

![image](https://user-images.githubusercontent.com/3605364/117202152-323e4a80-ad89-11eb-82d4-22b9bcbd4035.png)

Reply:
I put two complete versions in the file.